install.sh

```diff
@@ -6,7 +6,7 @@ set -e
 
 DOWNLOAD_URL="https://raw.githubusercontent.com/plutov/o/master/o.sh"
 TMP_FILE="/tmp/o.sh"
-BIN_FILE="/usr/local/bin/o"
+BIN_FILE="/usr/local/bin/"
```

Comment:
We can remove it now, as we set it on line 31.

Reply:
Done